Program's_Contributed_By_Contributors/C++ Programs/Array/Union_of_two_array.cpp

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
int freq[100002]={0};
    
    int doUnion(int a[], int n, int b[], int m)  {
        
        for(int i=0;i<n;i++){
            freq[a[i]]++;
        }
        for(int i=0;i<m;i++){
            freq[b[i]]++;
        }
        int count=0;
        for(int i=0;i<100002;i++){
            if(freq[i]>=1){
                count++;
            }
        }
        
        return count;
}